_hardcoded/tracing.go

```diff
@@ -160,9 +160,9 @@ func MuxServerMiddleware(serviceName string) func(http.Handler) http.Handler {
 // Used for setting up tracer provider
 func newResource() *resource.Resource {
 	var appName string
-	if os.Getenv("_POD_ID") != "" {
+	if os.Getenv("_APP_NAME") != "" {
```

Comment:
I know you were fixing a bug here so optional change. This can be simplified as
```go
if an := os.Getenv("_APP_NAME"); an != "" {
    appName = an
}
```

Reply:
That actually makes sense, but I want to leave the current formatting because I'm thinking I'll want to add more env vars to the resource later. Right now I'm doing the bare minimum.